component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/mobileconnect/MobileConnectAuthenticator.java

```diff
@@ -1174,6 +1218,44 @@ protected String getCallbackUrl(Map<String, String> authenticatorProperties) {
         return callbackURL;
     }
 
+    public boolean shouldPrefixClaimDialectUri() {
```

Comment:
Why can't you use protected instead ? . Is this intended to be called from outside other than inherited children ?

Reply:
There is no problem in this getting called from outside the inherited children. Thus no need to restrict access as protected.

Reply:
Is there any usage or usefulness of exposing this through authenticator ? It's ok to expose anything to outside. But we usually don't do it. We only expose it to outside only if it's required. If someone wants to find out this he can simply read configs and find out from outside. There is no need to call authenticator and get this. Hence better to make it protected.

Reply:
will do.